website/docs/guides/migration_guide_alert_conditions.html.markdown

```diff
@@ -12,7 +12,11 @@ Certain subtypes of Alert Conditions (Synthetics Alert Condition and Infra Alert
 
 Users wanting to migrate alert conditions will need to make a few adjustments to their configuration, by following the examples outlined below.
 
-### newrelic_synthetics_alert_condition
+### Migrating from Synthetics Alert Conditions to NRQL Alert Conditions
+
+The following example illustrates changing over to an NRQL-based alert condition using the [newrelic_nrql_alert_condition](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/nrql_alert_condition) resource from a Synthetics alert condition, i.e. [newrelic_synthetics_alert_condition](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/nrql_alert_condition).
+
+Below example also illustrates how argument monitor_id in synthetic alert condition should be inserted in NRQL over syntheticCheck after migration.
```

Comment:
```suggestion
### Migrating from Synthetics Alert Conditions to NRQL Alert Conditions

The following example illustrates changing over from a synthetics alert condition, i.e. [newrelic_synthetics_alert_condition](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/nrql_alert_condition) to an NRQL-based alert condition using the [newrelic_nrql_alert_condition](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/nrql_alert_condition) resource.

It may be obtained from the example that the ID of the monitor specified against the argument `monitor_id` in the synthetics alert condition may be used in NRQL with `SyntheticCheck` to convert this into an NRQL-based alert condition.
```

Reply:
Updated as suggested